keylime/cmd/ima_emulator_adapter.py

```diff
@@ -13,13 +13,16 @@
 from keylime.tpm.tpm_abstract import config, hashlib
 from keylime.common import algorithms
 
+# Instaniate tpm
+global tpm_instance
```

Comment:
Do we need to use a global for this so that the state is shared between them or the initialization happens at the beginning? Or is it to make sure that all these calls to `tpm()` send the right options? If the latter, then I think it's just as clear to pass those options to the calls, it's just a couple of places. Or maybe to create a new function that's called as a wrapper around `tpm()`. I know we use globals in various places already, but just looking at whether they are always required.